mesop/web/src/utils/base_path.ts

```diff
@@ -0,0 +1,13 @@
+export function prefixBasePath(path: string): string {
```

Comment:
Let's add a unit test for this function. 

If possible, let's see if we can add unit test coverage for some of the other changes, but only if it's not too hard to set up. Some places we don't have an existing coverage or the tests aren't easy to add. So I wouldn't put too much effort trying to add coverage. Mainly, just take a pragmatic approach.

I'd also suggest updating the Playwright config to test the base url changes. I think that would be more helpful than the unit tests. But off the top of my head not sure how to configure that, so that's more of a nice to have, but not necessary for this PR.

Thanks again.

Reply:
Sorry for the delayed response on this. I didn't realize you added the tests already.